charts/alfresco-search-enterprise/values.yaml

```diff
@@ -187,6 +187,9 @@ reindexing:
     pullPolicy: IfNotPresent
   pathIndexingEnabled: true
   environment: {}
+  # -- When to execute the reindexing job (e.g. `post-install` or `post-upgrade`
+  # or set to false to apply as a standard resource)
+  hookExecution: post-install
```

Comment:
I would also add a possibility to define https://helm.sh/docs/topics/charts_hooks/#hook-deletion-policies

Reply:
I preferred using `ttlSecondsAfterFinished` which applies anyway (even when not using hooks) and give a configurable amount of time to review logs, regardless of the outcome